test_libs/config_helpers/README.md

```diff
@@ -0,0 +1,19 @@
+# ETH 2.0 config helpers
+
+`preset_loader`: A util to load constants-presets with.
+See [Constants-presets documentation](../../configs/constants_presets/README.md).
+
+Usage:
+
+```python
+configs_path = 'configs/'
+
+...
+
+import preset_loader
+from eth2spec.phase0 import spec
+my_presets = preset_loader.load_presets(configs_path, 'main_net')
```

Comment:
There is 
```suggestion
my_presets = preset_loader.load_presets(configs_path, 'mainnet')
```

Reply:
Good spot, forgot this one when refactoring network type names.